x-pack/legacy/plugins/siem/server/lib/framework/kibana_framework_adapter.ts

```diff
@@ -27,11 +27,13 @@ interface CallWithRequestParams extends GenericParams {
 
 export class KibanaBackendFrameworkAdapter implements FrameworkAdapter {
   public version: string;
-  public envMode: EnvironmentMode;
+  private isProductionMode: boolean;
+  private router: IRouter;
 
-  constructor(private server: ServerFacade, mode: EnvironmentMode) {
-    this.version = server.config().get('pkg.version');
-    this.envMode = mode;
+  constructor(core: CoreSetup, private __legacy: ServerFacade) {
+    this.version = __legacy.config().get('pkg.version');
+    this.isProductionMode = process.env.NODE_ENV === 'production';
```

Comment:
Please let me know if there's a better way to determine this; there doesn't seem to be an equivalent `EnvironmentMode` in the new platform.

Reply:
Nope, I'm wrong, it's still there. I'll change this back to use `env.mode.prod` when I move this code into the plugin itself.